integration/integration-api/build.gradle

```diff
@@ -0,0 +1,10 @@
+dependencies {
```

Comment:
I think we can remove a lot of these dependencies, right? Maybe even all?

Reply:
All of them have been removed